src/run/run_step.go

```diff
@@ -147,6 +150,13 @@ func environ(config *core.Configuration, setenv bool) []string {
 	return env
 }
 
+// adRunEnviron returns values that are appended to the environment for a command.
+func adRunEnviron() []string {
```

Comment:
probably make this just a top-level variable instead of a function.

also it should be camelCase for Go...

Reply:
Discussed offline in [gitter](https://gitter.im/please-build/Lobby?at=5d128b9a527a392800c1103e).

> I wanted to keep a separate state-bit for "This directory is ready for use", which lives outside the directory itself.